### DISCUSSION TITLE
KeyError: 'model.diffusion_model.input_blocks.0.0.weight'

### DISCUSSION THREAD

**Discussion by Aloukik21** [lllyasviel/stable-diffusion-webui-forge#576] [General]
Getting this error after loading SVD (linux)

**Comment by waicharikim** [comment]
facing the same issue, any resolution?

**Comment by Jemno23** [comment]
Also having the same issue (Windows) 

"Error occurred when executing CheckpointLoaderSimple:

'model.diffusion_model.input_blocks.0.0.weight'"

**Comment by Kettuu** [comment]
Same error here. Trying to create a pic with "Fooocus" and using my own Lora model.
